src/components/BottomMenu.tsx

```diff
@@ -119,13 +152,7 @@ export function BottomMenu({
           target="_blank"
           rel="noopener noreferrer"
         >
-          {formatMessage(
-            viewState === 'CompleteOptionsInverted'
-              ? bottomMenuMessages.showPolicyButtonCompleteOptionsInverted
-              : viewState === 'AcceptAllOrMoreChoices'
-              ? bottomMenuMessages.showPolicyButtonsAcceptAllOrMoreChoices
-              : bottomMenuMessages.showPolicyButtonPrimary,
-          )}
+          {formatMessage(VIEW_STATE_TO_MESSAGE[viewState])}
```

Comment:
even though we have the viewState typed here, i'd probably add some defensive code in case its improperly set some how.

earlier up in the file:
`const message = VIEW_STATE_TO_MESSAGE[viewState]`

then when rendering:
`{message ? formatMessage(message) : undefined}`